demo/src/main/java/com/novoda/storagepathfinder/demo/FileSizeFinder.java

```diff
@@ -0,0 +1,66 @@
+package com.novoda.storagepathfinder.demo;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+
+final class FileSizeFinder {
+
+    private FileSizeFinder() {
+        // Uses static factory methods.
```

Comment:
Not really factory, just static (or utility) ones

Reply:
😂 this was automatic, I've done so many recently 😂

Reply:
8753c1e